sunpy/net/dataretriever/tests/test_eve.py

```diff
@@ -100,3 +100,8 @@ def test_levels(time):
     qr = Fido.search(time, eve_a, a.Level(0) | a.Level(1))
     clients = {type(a.client) for a in qr.responses}
     assert clients.symmetric_difference({VSOClient, eve.EVEClient}) == set()
+
+
+def test_attr_reg():
+    a.Instrument.eve = a.Instrument('EVE')
+    a.Level.A0 = a.Level('0')
```

Comment:
I get we need to prepend something before the number but why `A`? Can we make this prefix attr specific? `L` would make more sense for level?

Reply:
This test is wrong, it should be:
```
a.Level.zero = a.Level('0')
```
Is that better?

Reply:
yeah nice.